jetty-ee9/jetty-ee9-nested/src/main/java/org/eclipse/jetty/ee9/nested/HttpChannel.java

```diff
@@ -1602,4 +1602,20 @@ public void dispatch() throws IOException, ServletException
             _request.setHandled(true);
         }
     }
+
+    private class NeedContentInvocableTask implements Invocable.Task
```

Comment:
`DemandTask extends readyTask`.

Reply:
In this case we cannot use ReadyTask, as the invocation type will not be known at the time of construction.